git_machete/client.py

```diff
@@ -707,7 +707,7 @@ def traverse(
 
             if branch != current_branch and (needs_slide_out or needs_parent_sync or needs_remote_sync):
                 self.__print_new_line(False)
-                sys.stdout.write(f"Checking out {bold(branch)}\n")
+                print(f"Checking out {bold(branch)}\n")
```

Comment:
Hmm is the behavior exactly the same as before the change here? doesn't `print` append `\n` automatically, unlike `sys.stdout.write`? 🤔